apps/nextjs-website/src/pages/[productSlug]/overview.tsx

```diff
@@ -17,6 +17,8 @@ export type OverviewPageProps = {
   readonly path: string;
   readonly product: Product;
   readonly hero: {
+    readonly backgroundImage: string;
+    readonly altText?: string;
```

Comment:
```suggestion
    readonly altText: string;
```
a key factor for this developer portal is accessibility, so if the `backgroundImage` is not optional, I think nor the `altText` should be